sheep/src/lib.rs

```diff
@@ -1,3 +1,6 @@
+#![feature(test)]
+extern crate test;
```

Comment:
Benchmarks should better go into a `benches` folder, otherwise this doesn't compile on stable.

Reply:
right, thanks!